blog/09-05-2023-smart-contract-security-on-celo-with-securify/smart-contract-security-on-celo-with-securify.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Smart contract security on Celo with Securify
+description: Analyze smart contracts using free security tools - Slither
+authors:
+  - name: Isaac Jesse
+    title: Web3, Smart Contract Developer
+    url: https://github.com/bobeu
+    image_url: https://github.com/bobeu.png
+tags: [celosage, intermediate, smartcontract, solidity]
+hide_table_of_contents: true
+slug: /tutorials/smart-contract-security-on-celo-with-securify
+---
+
+![header](../../src/data-tutorials/showcase/intermediate/smart-contract-security-on-celo-with-securify.png)
+
+## Introduction
+
+Smart contracts are self-executing pieces of code deployed on a blockchain. EVM-compatible blockchains such as Celo are free from censorship, downtime, and third-party interference, and have since invented changed how software is perceived. However, as exciting as smart contracts are, they introduce known and probably undetected risks ranging from low to critical in severity. It is crucial for developers to thoroughly scrutinize contract code before deployment.
+
+## Prerequisites​
+This guide introduces you to working with a security tool called Slither. You must at least be an intermediate working with smart contracts using solidity. If you're new in this space, I recommend starting from **[here](https://docs.celo.org/blog/tutorials/building-a-solidity-smart-contract-for-nft-royalty-fees-a-step-by-step-guide)** and **[here](https://docs.celo.org/blog/tutorials/best-practices-for-writing-smart-contracts-with-real-world-examples)**.
+
+## Requirements​
+
+The following tools are recommended and should be installed before proceeding.
+
+- [Python](https://python.org)
```

Comment:
Maybe you can be more specific about the Python 3 versions that are compatible with this tool.

Reply:
> Maybe you can be more specific about the Python 3 versions that are compatible with this tool.

There is no specific version. I mentioned the range of version, and I used python3. 8 as an example. I think that's sufficient